tests/nightly/nightly-autorun.sh

```diff
@@ -16,8 +16,10 @@
 
 # Get the latest code from vmware/vic repo
 cd ~/go/src/github.com/vmware/vic
-git fetch
+git clean -fd
```

Comment:
I think there's a nightly secrets file in the project's directory and doing this will remove it.